prime-angular-frontend/src/app/modules/enrolment/pages/regulatory/regulatory.component.ts

```diff
@@ -120,8 +124,10 @@ export class RegulatoryComponent extends BaseEnrolmentProfilePage implements OnI
       return;
     }
 
+    const enrolment = this.enrolmentService.enrolment;
+
     // Replace previous values on deactivation so updates are discarded
-    this.formState.patchValue(this.enrolmentService.enrolment.certifications);
+    this.formState.patchValue({ certifications: enrolment.certifications, deviceProviderIdentifier: enrolment.deviceProviderIdentifier });
```

Comment:
Similar to the other updates this can be destructured.  It makes it easier to read and reduces line length.

Reply:
refactored